src/library/useUnsavedDirtyFormDataUtilities.js

```diff
@@ -27,14 +27,6 @@ export const useUnsavedDirtyFormDataUtilities = (sessionStorageName) => {
     }
   }, [sessionStorageName])
 
-  const _clearPersistedUnsavedFormDataBeforeUnload = useEffect(() => {
```

Comment:
this function was breaking things so I removed it. unless someone has a good case for why it's necessary, i'm keeping it out

Reply:
oh, quick comment before a meeting, but I think this is so that if a user navigates away, we want to clear out the persist data, otherwise the dirty form will load with stale data when the user navigates back. We only want to persist as a hack for network or whatever other react refreshes are caused by prop changes that may clear the dirty form

Reply:
yeah, that is totally going opposite of what you are doing.